_posts/2015-03-27-ApiBlueprint-button.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Sharing is caring - socialize you're public API blueprint
+excerpt: Learn the art of API blueprint through socialization and collaboration.
+layout: post
+date: 2015-03-27 00:00:00 +0100
+author: vincenzo
+published: true
+commets: true
+---
+
+We have recently received a lot of feedback from users that would like to learn more about API Blueprint and how to present stunning and beautiful documentation.
+
+We care about your feedback, and we think that documentation and tutorials about [API Blueprint](http://apiblueprint.org) are extremely important.
+
+One of the most requested features was to have the ability to look to the other blueprints as a source of inspiration, and today this is possible for all public API Blueprints.
+
+Whenever you're surfing a public blueprint, a new **API Blueprint** button will appear in header bar. Clicking this button will open up a new window with readable markdown, that can be easily grabbed, studied, modified and republished.
```

Comment:
Mmm...I do not think so. It depends on your **clone** meaning.
If you mean "create a new api under my account with that blueprint as starting point", then no.